gnomad/sample_qc/pipeline.py

```diff
@@ -149,6 +150,7 @@ def get_qc_mt(
     :param filter_exome_low_coverage_regions: If set, only high coverage exome regions (computed from gnomAD are kept)
```

Comment:
I know this wasn't put in by you, but while you are modifying this function can you add a period after each of these parameter descriptions?